operate/qa/integration-tests/pom.xml

```diff
@@ -47,6 +47,12 @@
       <groupId>io.camunda</groupId>
       <artifactId>camunda-zeebe</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.springframework.data</groupId>
+          <artifactId>spring-data-jpa</artifactId>
```

Comment:
This is needed because with the dependency for some reason the tests were failing because classes were getting constructed vs mocked. For now I think this is fine though